backend/selenium_tests/page_object/people/people.py

```diff
@@ -0,0 +1,133 @@
+    individualAge = 'th[data-cy="individual-age"]'
+    individualSex = 'th[data-cy="individual-sex"]'
+    individualLocation = 'th[data-cy="individual-location"]'
+    tableRow = 'tr[data-cy="table-row"]'
+    tablePagination = 'div[data-cy="table-pagination"]'
+    individualTableRow = 'tr[data-cy="individual-table-row"]'
+
+    def getPageHeaderContainer(self) -> WebElement:
+        return self.wait_for(self.pageHeaderContainer)
+
+    def getPageHeaderTitle(self) -> WebElement:
+        return self.wait_for(self.pageHeaderTitle)
+
+    def getIndFiltersSearch(self) -> WebElement:
+        return self.wait_for(self.indFiltersSearch)
+
+    def getSelectFilter(self) -> WebElement:
+        return self.wait_for(self.selectFilter)
+
+    def getFiltersDocumentType(self) -> WebElement:
+        return self.wait_for(self.filtersDocumentType)
+
+    def getFiltersDocumentNumber(self) -> WebElement:
+        return self.wait_for(self.filtersDocumentNumber)
+
+    def getIndFiltersAdmin1(self) -> WebElement:
+        return self.wait_for(self.indFiltersAdmin1)
+
+    def getAdminLevel1Input(self) -> WebElement:
+        return self.wait_for(self.adminLevel1Input)
+
+    def getIndFiltersAdmin2(self) -> WebElement:
+        return self.wait_for(self.indFiltersAdmin2)
+
+    def getAdminLevel2Input(self) -> WebElement:
+        return self.wait_for(self.adminLevel2Input)
+
+    def getIndFiltersGender(self) -> WebElement:
+        return self.wait_for(self.indFiltersGender)
+
+    def getIndFiltersAgeFrom(self) -> WebElement:
+        return self.wait_for(self.indFiltersAgeFrom)
+
+    def getIndFiltersAgeTo(self) -> WebElement:
+        return self.wait_for(self.indFiltersAgeTo)
+
+    def getIndFiltersFlags(self) -> WebElement:
+        return self.wait_for(self.indFiltersFlags)
+
+    def getIndFiltersOrderBy(self) -> WebElement:
+        return self.wait_for(self.indFiltersOrderBy)
+
+    def getIndFiltersStatus(self) -> WebElement:
+        return self.wait_for(self.indFiltersStatus)
+
+    def getIndFiltersRegDateFrom(self) -> WebElement:
+        return self.wait_for(self.indFiltersRegDateFrom)
+
+    def getIndFiltersRegDateTo(self) -> WebElement:
+        return self.wait_for(self.indFiltersRegDateTo)
+
+    def getButtonFiltersClear(self) -> WebElement:
+        return self.wait_for(self.buttonFiltersClear)
+
+    def getButtonFiltersApply(self) -> WebElement:
+        return self.wait_for(self.buttonFiltersApply)
+
+    def getPageDetailsContainer(self) -> WebElement:
+        return self.wait_for(self.pageDetailsContainer)
+
+    def getTableTitle(self) -> WebElement:
+        return self.wait_for(self.tableTitle)
+
+    def getSanctionListPossibleMatch(self) -> WebElement:
+        return self.wait_for(self.sanctionListPossibleMatch)
+
+    def getTableLabel(self) -> WebElement:
+        return self.wait_for(self.tableLabel)
+
+    def getIndividualId(self) -> WebElement:
+        return self.wait_for(self.individualId)
+
```

Comment:
How about something like: 
```
class YourClass:
    def __init__(self):
        self.elements = {
            "getPageHeaderContainer": self.pageHeaderContainer,
            "getPageHeaderTitle": self.pageHeaderTitle,
            "getIndFiltersSearch": self.indFiltersSearch,
            "getSelectFilter": self.selectFilter,
            "getFiltersDocumentType": self.filtersDocumentType,
            "getFiltersDocumentNumber": self.filtersDocumentNumber,
            "getIndFiltersAdmin1": self.indFiltersAdmin1,
            "getAdminLevel1Input": self.adminLevel1Input,
            "getIndFiltersAdmin2": self.indFiltersAdmin2,
            "getAdminLevel2Input": self.adminLevel2Input,
            "getIndFiltersGender": self.indFiltersGender,
            "getIndFiltersAgeFrom": self.indFiltersAgeFrom,
            "getIndFiltersAgeTo": self.indFiltersAgeTo,
        }

    def get_element(self, name: str) -> WebElement:
        return self.wait_for(self.elements[name])
```

 this could reduce the amount of code. Then, instead of calling getPageHeaderContainer(), you would call get_element("getPageHeaderContainer").